src/components/Breadcrumb/Breadcrumb.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { withKnobs, boolean, array, select, text } from "@storybook/addon-knobs";
+import Breadcrump from './Breadcrumb';
+import Stack from '../storyUtils/Stack';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from "react-router-dom";
+
+<Meta title="Design System|Breadcrumb" component={() => true} />
+
+# Breadcrumb
+
+Description of Breadcrumb component
+
+# Usage 1
+
+```js
+import { Breadcrumb } from 'orfium-design-system';
+
+<Breadcrumb>
+  <div>content 1</div>
+  <div>content 2</div>
+  <div>content 3</div>
+</Breadcrumb>
+```
+
+# Usage 2
+
+```js
+import { Breadcrumb } from 'orfium-design-system';
+
+<Breadcrumb data=[{to:"/path", label: "label"}]/>
+```
+
+# Props
+
+<Props of={() => true} />
+
+
+# Simple Breadcrumbs (up to 3 levels)
+A simple Breadcrumb example
+
+<Preview>
+  <Story name="Simple Breadcrumbs">
+    <Stack>
+        <Breadcrump>
+          <div>First Level</div>
+          <div>Second Level</div>
+          <div>Third Level</div>
+          <div>Forth Level</div>
+          <div>Current Level</div>
+        </Breadcrump>
+    </Stack>
+  </Story>
+</Preview>
+
+# Simple Breadcrumbs (more than 3 levels)
+A simple Breadcrumb with n levels example
+
+<Preview>
+  <Story name="Simple Breadcrumbs with options">
+    <Stack>
+      <Router>
+          <Breadcrump data={[
+            { to: '/', label: 'Home' },
+            { to: '/first-level', label: 'First Level' },
+            { to: '/second-level', label: 'Second Level' },
+            { to: '/third-level', label: 'Third Level' },
+            { to: '/forth-level', label: 'Forth Level' },
+          ]}/>
+          <Switch>
+                <Route path="/first-level">
+                  {() => <div>first level</div>}
+                </Route>
+                <Route path="/second-level">
+                  {() => <div>second level</div>}
+                </Route>
+                <Route path="/third-level">
+                  {() => <div>third level</div>}
+                </Route>
+                <Route path="/forth-level">
+                  {() => <div>forth level</div>}
+                </Route>
+          </Switch>
+      </Router>
+    </Stack>
+  </Story>
+ 
+</Preview>
+
+# Advanced Breadcrumbs (with dd in levels)
+A advanced Breadcrumb with dd levels example
```

Comment:
is this finished ?

Reply:
This is not implemented, I would like to further discuss the approach of the implementation.